.gitlab-ci.yml

```
@@ -204,28 +215,28 @@ sanitizer thread:
       *) gdb -q -ex "handle SIG32 nostop" -ex r -ex bt -ex q --return-child-result --args $command ;;
       esac
 
-sanitizer address test:
+Linux Address sanitizer test:
```

```suggestion
Linux (Address sanitizer) test:
```

nit: maybe it might look better if they matched what they "need"?